packages/frontend/src/pages/Contribute.jsx

```diff
@@ -467,11 +467,7 @@ export default observer(() => {
                             fontWeight: '600',
                           }}
                           onClick={async () => {
-                            try {
-                              await ceremony.join(name, 'open')
-                            } catch (e) {
-                              setError(e)
-                            }
+                            await ceremony.join(name, 'open')
```

Comment:
Why remove the try&catch here?

Reply:
The [`Button`](https://github.com/Unirep/ceremony/blob/main/packages/frontend/src/components/Button.jsx#L7) component will catch the error and display the message. The `setError` handler in this component doesn't properly handle `Error` objects, the check for `error.length` always returns `undefined` in the hook.